generators/server/templates/app/src/main/resources/logback-spring.xml

```diff
@@ -10,6 +10,13 @@
     </springProfile>
     <springProfile name="!default">
         <include resource="org/springframework/boot/logging/logback/file-appender.xml" />
+        <!-- definition of appender STDOUT -->
```

Comment:
console-appender is already included above using <include> directive.

Reply:
When I run the code generated using gradle in windows I am getting below error

```
java.lang.IllegalStateException: Logback configuration error detected: 
ERROR in ch.qos.logback.core.joran.action.AppenderRefAction - Could not find an appender named [STDOUT]. Did you define it below instead of above in the configuration file?
```

Hence I have added definition of STDOUT to make it run.

Reply:
Yes, the `console-appender.xml` defines the appender with name as **CONSOLE**, but it is configured as **STDOUT** which is wrong. We just need to update it to **CONSOLE**, that should fix it.

Reply:
Updated PR with CONSOLE